packages/manager/apps/okms/src/common/utils/tests/uiTestHelpers.ts

```diff
@@ -60,6 +60,9 @@ const getOdsButton = async ({
     },
     { ...WAIT_FOR_DEFAULT_OPTIONS, ...options },
   );
+  if (!button) {
+    throw new Error(`Button ${label ?? altLabel ?? iconName ?? 'not found'} not found`);
```

Comment:
To avoid "Button not found not found"
```suggestion
    throw new Error(`Button ${label ?? altLabel ?? iconName ?? ''} not found`);
```